packages/web3/src/token/nft.test.ts

```diff
@@ -18,116 +18,127 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 
 import { validateNFTCollectionUriMetaData, validateNFTTokenUriMetaData, validateNFTBaseUri } from './nft'
 
-describe('nft', function () {
-  it('should validate NFT and NFT collection metadata', () => {
-    const validWithoutAttributes = {
+describe('NFT Metadata Validation', () => {
+  describe('Token URI Metadata', () => {
+    const validMetadataWithoutAttributes = {
       name: 'NFT name',
       description: 'NFT description',
       image: 'https://example.com/'
     }
 
-    expect(validateNFTTokenUriMetaData(validWithoutAttributes)).toEqual(validWithoutAttributes)
-    expect(validateNFTCollectionUriMetaData(validWithoutAttributes)).toEqual(validWithoutAttributes)
-
-    const validWithAttributes = {
+    const validMetadataWithAttributes = {
       name: 'NFT name',
       description: 'NFT description',
       image: 'https://example.com/',
       attributes: [
-        {
-          trait_type: 'color',
-          value: 'blue'
-        },
-        {
-          trait_type: 'size',
-          value: 100
-        },
-        {
-          trait_type: 'valid',
-          value: true
-        }
+        { trait_type: 'color', value: 'blue' },
+        { trait_type: 'size', value: 100 },
+        { trait_type: 'valid', value: true }
       ]
     }
 
-    expect(validateNFTTokenUriMetaData(validWithAttributes)).toEqual(validWithAttributes)
-    expect(() => validateNFTCollectionUriMetaData(validWithAttributes)).toThrow(Error)
-
-    const withEmptyDescription = {
-      name: 'NFT name',
-      description: '',
-      image: 'https://example.com/'
-    }
-    expect(() => validateNFTTokenUriMetaData(withEmptyDescription)).toThrow(Error)
-    expect(() => validateNFTCollectionUriMetaData(withEmptyDescription)).toThrow(Error)
-
-    const withEmptyName = {
-      name: '',
-      description: 'NFT description',
-      image: 'https://example.com/'
-    }
-    expect(() => validateNFTTokenUriMetaData(withEmptyName)).toThrow(Error)
-    expect(() => validateNFTCollectionUriMetaData(withEmptyName)).toThrow(Error)
-
-    const withWrongField = {
-      name: 'NFT name',
-      description: '',
-      url: 'https://example.com/'
-    }
-    expect(() => validateNFTTokenUriMetaData(withWrongField)).toThrow(Error)
-    expect(() => validateNFTCollectionUriMetaData(withWrongField)).toThrow(Error)
+    it('validates metadata without attributes', () => {
+      expect(validateNFTTokenUriMetaData(validMetadataWithoutAttributes)).toEqual(validMetadataWithoutAttributes)
+    })
+
+    it('validates metadata with attributes', () => {
```

Comment:
I am not sure if it adds that much clarity with `it('description....` around every tests, it only makes the diff unnecessarily large.

e.g. it is clear that this tests the case where we have empty description.
```
expect(() => validateNFTTokenUriMetaData(withEmptyDescription)).toThrow(Error)
```

Is it possible to not change the structure of the existing tests and only add/update more tests?